etc/profile-a-l/Beyond-All-Reason.profile

```diff
@@ -0,0 +1,42 @@
+# Firejail profile for Beyond All Reason
+# Description: Open source real time strategy game
+# This file is overwritten after every install/update
+# Persistent local customizations
+include beyondallreason.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/Beyond All Reason
+noblacklist ${HOME}/Downloads
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
```

Comment:
Can we add `include disable-exec.inc`?

Reply:
Game cannot start when this is enabled. It downloads game patches to the folder with the home directory and then loads them (EACCESS if disable-exec is enabled)

Reply:
IN that case add `ignore noexec ${HOME}`.